### PR TITLE
[ENHANCEMENT] Update all the versions!

### PR DESCRIPTION
**Pull request by seawatts**
* Update ember to 2.5.0
* Update ember-cli to 2.5.0 (ran ember init)
* Remove unused dependencies
* Updated to use more es6 style syntax
* Fix tests by removing bind-attr

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,17 +11,14 @@ cache:
 
 env:
   - EMBER_TRY_SCENARIO=default
-  - EMBER_TRY_SCENARIO=1.10.1
-  - EMBER_TRY_SCENARIO=1.11.3
-  - EMBER_TRY_SCENARIO=1.12.1
+  - EMBER_TRY_SCENARIO=ember-1.13
   - EMBER_TRY_SCENARIO=ember-release
   - EMBER_TRY_SCENARIO=ember-beta
   - EMBER_TRY_SCENARIO=ember-canary
```
Comment (lukemelia) [inline .travis.yml]: I would like to maintain compatibility with Ember 1.10+ unless there is a compelling reason to drop it.

File: ember-cli-build.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
+/*jshint node:true*/
 /* global require, module */
-var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
-module.exports = function(/*defaults*/) {
-  var app = new EmberApp({
-    snippetSearchPaths: ['tests/dummy/app']
```
Comment (lukemelia) [inline ember-cli-build.js]: Is deletion of this OK for some reason? I believe it exists for the dummy app code snippet rendering.

**Comment by lukemelia** [comment]
Thanks @seawatts! I left some comments, above. If you can address those, I think we can merge. Probably good for @chrislopresto to have a look too.

**Comment by chrislopresto** [comment]
Fixed up by https://github.com/yapplabs/ember-wormhole/pull/53

**Pull request closed without merging**